content/en/real_user_monitoring/reactnative/integrated_libraries.md

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
Does this page also needs to be added in `config/_default/menus/menus.en.yaml` to be referenced?

Reply:
yes, yes it does :) adding it!